rllib/core/rl_module/torch/torch_rl_module.py

```diff
@@ -27,6 +29,56 @@ def get_state(self) -> Mapping[str, Any]:
     def set_state(self, state_dict: Mapping[str, Any]) -> None:
         self.load_state_dict(state_dict)
 
+    @override(RLModule)
+    def save_to_checkpoint(self, checkpoint_dir_path: str) -> None:
+        """Saves the module to a checkpoint directory.
+
+        Args:
+            dir_path: The directory to save the checkpoint to.
+
+
+        Raises:
+            ValueError: If dir_path is not an absolute path.
+        """
+        path = pathlib.Path(checkpoint_dir_path)
+        if not path.is_absolute():
+            raise ValueError("dir_path must be an absolute path.")
+        path.mkdir(parents=True, exist_ok=True)
+        module_state_path = path / "module_state.pt"
+        torch.save(self.state_dict(), str(module_state_path))
+        self._save_module_metadata(path, module_state_path)
+
+    @classmethod
+    def load_from_checkpoint(cls, checkpoint_dir_path: str) -> "TorchRLModule":
+        """Loads the module from a checkpoint directory.
+
+        Args:
+            dir_path: The directory to load the checkpoint from.
+        """
+        path = pathlib.Path(checkpoint_dir_path)
+        if not path.exists():
+            raise ValueError(
+                "While loading from checkpoint there was no directory"
+                " found at {}".format(checkpoint_dir_path)
+            )
+        if not path.is_absolute():
+            raise ValueError("dir_path must be an absolute path.")
+        if not path.is_dir():
+            raise ValueError(
+                "While loading from checkpoint the checkpoint_dir_path "
+                "provided was not a directory."
+            )
+
+        module = cls._from_metadata_file(path)
+        state_path = path / "module_state.pt"
```

Comment:
we should check this before loading the module. FAIL FAST :)

Reply:
done